tests/test_unit/test_prerun_transformations/test_transformations.py

```diff
@@ -0,0 +1,633 @@
+    # SumTimseries with only input_1
+    # SumTimseries with input_1 and input_2
+    # SumTimseries with input_1, input_2, and input_3
+    for length in range(len(input_data)):
+        expected_dates = pd.concat(input_dates[: length + 1]).sort_values().drop_duplicates()
+        expected_data = pd.Series(all_expected_data[length], index=expected_dates)
+
+        transformation = SumTimeseries()
+        output_data = transformation.apply(time_series_data=input_data[: length + 1])
+
+        pd.testing.assert_series_equal(expected_data, output_data, check_dtype=False, check_freq=False)
+
+
+STATIC_VALUES_TEST_CASES = [
+    # StaticValues test cases
+    TransformationTestCase(
+        case_id="StaticValues: case_1",
+        transformation=StaticValues(
+            relative_datapoints=[
+                RelativeDatapoint(offset_minute=0, offset_value=42),
+                RelativeDatapoint(offset_minute=60, offset_value=420),
+                RelativeDatapoint(offset_minute=1440, offset_value=4200),
+            ]
+        ),
+        expected_values=[42, 420, 4200],
+        expected_times=[
+            datetime(2022, 1, 1, 0, tzinfo=None),
+            datetime(2022, 1, 1, 1, tzinfo=None),
+            datetime(2022, 1, 2, 0, tzinfo=None),
+        ],
+    ),
+    TransformationTestCase(
+        case_id="StaticValues: case_2",
+        transformation=StaticValues(
+            relative_datapoints=[
+                RelativeDatapoint(offset_minute=0, offset_value=42),
+                RelativeDatapoint(offset_minute=1, offset_value=-420),
+                RelativeDatapoint(offset_minute=-120, offset_value=4200),  # TODO: do we want to allow this?
+            ]
+        ),
+        expected_values=[42, -420, 4200],
+        expected_times=[
+            datetime(2022, 1, 1, 0, tzinfo=None),
+            datetime(2022, 1, 1, 0, 1, tzinfo=None),
+            datetime(2021, 12, 31, 22, tzinfo=None),
+        ],
+    ),
+]
+
+
+@pytest.mark.parametrize(
+    "test_case",
+    [pytest.param(test_case, id=test_case.case_id) for test_case in STATIC_VALUES_TEST_CASES],
+)
+def test_static_values(cognite_client_mock: CogniteClient, test_case: TransformationTestCase):
+
+    test_case.transformation.pre_apply(
+        client=cognite_client_mock, shop_model={}, start=test_case.start_date, end=test_case.start_date
+    )
+    output_data = test_case.transformation.apply((pd.Series(),))
+
+    expected_data = pd.Series(test_case.expected_values, index=test_case.expected_times)
+
+    pd.testing.assert_series_equal(expected_data, output_data, check_dtype=False, check_freq=False)
+
+
+def test_height_to_volume(cognite_client_mock: CogniteClient):
+    heights = [2, 4, 6, 8, 10]
+    volumes = [10, 20, 40, 80, 160]
+
+    datapoints = pd.Series(
+        {
+            1: 1,  # below interpolation bounds
+            2: 4,
+            3: 6,
+            4: 7,  # interpolated
+            5: 11,  # above interpolation bounds
```

Comment:
Interesting one - not sure if this should be allowed, and what the correct output should be if we allow it (but just using the highest seems like an OK option)